e2e/packaging_test.go

```diff
@@ -22,20 +22,24 @@ import (
 
 	"github.com/ossf/scorecard/v2/checker"
 	"github.com/ossf/scorecard/v2/checks"
+	"github.com/ossf/scorecard/v2/clients/githubrepo"
 	scut "github.com/ossf/scorecard/v2/utests"
 )
 
 var _ = Describe("E2E TEST:Packaging", func() {
```

Comment:
use the `check.CheckPackaging` instead of hardcoding the string "Packaging"

Reply:
Done.